langstream-runtime/langstream-runtime-impl/src/main/python/langstream_grpc/api.py

```diff
@@ -63,12 +63,28 @@ def headers(self) -> List[Tuple[str, Any]]:
 RecordType = Union[Record, dict, list, tuple]
 
 
+class TopicProducer(ABC):
+    """The topic producer interface"""
+
+    async def write(self, topic: str, record: Record):
+        """Write a record to a topic (for async methods)."""
+        pass
+
+    def write_future(self, topic: str, record: Record) -> Future:
```

Comment:
Suggestions for a better name ?

Reply:
what about "awrite" for the async write and "write" for the sync version ?

Reply:
Yes, I've thought of that. Now both methods are kinda "async", one with async/await, the other with Future.

Reply:
But let's use `awrite` as it's a defacto convention when 2 methods do the same thing and one does it on the event loop.